pkgs/misc/lilypond/default.nix

```
@@ -19,9 +19,10 @@ stdenv.mkDerivation rec {
     for f in "$out/bin/"*; do
         # Override default argv[0] setting so LilyPond can find
         # its Scheme libraries.
-        wrapProgram "$f" --set GUILE_AUTO_COMPILE 0 \
-                         --set PATH "${ghostscript}/bin" \
-                         --argv0 "$f"
+        wrapProgram "$f" \
+          --set GUILE_AUTO_COMPILE 0 \
+          --set PATH "${lib.makeBinPath [ ghostscript coreutils "$out" ]}" \
```

```suggestion
          --set PATH "${lib.makeBinPath [ ghostscript coreutils (placeholder "out") ]}" \
```

Maybe this is better? Dealing with bash variables in Nix functions is dangerous.

---

> Dealing with bash variables in Nix functions is dangerous.

We are doing this all over the place.